sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -36,15 +36,15 @@ function createZumoNamespace() {
         this.canRunUnattended = true;
         this.logs = [];
         this.requiredFeatures = [];
-        if (requiredRuntimeFeatures) {
-            if (typeof requiredRuntimeFeatures === 'string') {
-                requiredRuntimeFeatures = [requiredRuntimeFeatures];
+        if (requiredRuntimeFeatures) {
+            if (typeof requiredRuntimeFeatures === 'string') {
+                requiredRuntimeFeatures = [requiredRuntimeFeatures];
             }
             var that = this;
-            requiredRuntimeFeatures.forEach(function (feature) {
-                that.requiredFeatures.push(feature);
-            });
-        }
+            requiredRuntimeFeatures.forEach(function (feature) {
+                that.requiredFeatures.push(feature);
+            });
```

Comment:
Another nit / FYI:

Above four lines can be written as:

    this.requiredFeatures.push.apply(this.requiredFeatures, requiredRuntimeFeatures)

or

    Array.prototype.push.apply(this.requiredFeatures, requiredRuntimeFeatures)

If you wanted really concise code, you could collapse the above 7 lines into:

    Array.prototype.push.apply(this.requiredFeatures, requiredRuntimeFeatures.constructor !== Array ? [requiredRuntimeFeatures] : requiredRuntimeFeatures)

Though keeping the array test separate is more readable.